assets/css/tasks/_contact-tasks.css

```diff
@@ -0,0 +1,71 @@
+/* CONTACT PAGE */
+
+.contact-section {
+    background-image: url(/img/contact-img.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right 10rem bottom 0;   
+}
+.form {
+    padding: 20px;
+    width: 40%;
+    background-color: #f3f2fc;
+    border-radius: 1rem;
+    position: left;
```

Comment:
The "position" property is given the value:
`` ''
position: relative / absolute / etc.
`` ''
And then we can set the dimension value of this position. E.g:
`` ''
on the left: 1rem;
on the right: 1rem;
top: 1rem;
bottom: 1rem;
`` ''
So now, do you know how to improve it?

Reply:
Thanks!